src/main/java/com/hillel/items_exchange/model/Location.java

```diff
@@ -22,8 +22,8 @@
     private String city;
 
     @Enumerated(EnumType.STRING)
-    @Column(name = "lang")
-    private Lang lang;
+    @Column(name = "i18n")
```

Comment:
You could not to use annotation Column if the names of the column in DB and into entity are absolutely equal

Reply:
Fixed